src/Database/Driver/Sqlserver.php

```diff
@@ -64,8 +64,14 @@ public function connect()
             return true;
         }
         $config = $this->_config;
+
+        // PDO::ATTR_PERSISTENT is not allowed by SQL Server PHP drivers.
+        // See: https://github.com/Microsoft/msphpsql/issues/65
+        if (isset($config['persistent']) && $config['persistent']) {
+            throw new \InvalidArgumentException('Sqlserver driver does not allow config setting: persistent');
```

Comment:
I don't think we should tell anyone to set it to false becaue it may give the impression that we are telling the underlying driver to do something related to persistence of connections.  We are not.

Connection persistence can only be manipulated through the configuration of the underlying ODBC layer, which is not exposed through the PHP PDO drivers.